python/cudf/cudf/testing/testing.py

```diff
@@ -568,40 +614,62 @@ def assert_frame_equal(
         left, right = left.reindex(index=right.index), right
         right = right[list(left._data.names)]
 
-    if check_less_precise:
-        raise NotImplementedError("check_less_precise is not yet supported")
-
     # index comparison
-    assert_index_equal(
-        left.index,
-        right.index,
-        exact=check_index_type,
-        check_names=check_names,
-        check_less_precise=check_less_precise,
-        check_exact=check_exact,
-        check_categorical=check_categorical,
-        obj=f"{obj}.index",
-    )
+    if PANDAS_GE_110:
+        assert_index_equal(
+            left.index,
+            right.index,
+            exact=check_index_type,
+            check_names=check_names,
+            check_exact=check_exact,
+            check_categorical=check_categorical,
+            rtol=rtol,
+            atol=atol,
+            obj=f"{obj}.index",
+        )
+    else:
+        assert_index_equal(
+            left.index,
+            right.index,
+            exact=check_index_type,
+            check_names=check_names,
+            check_exact=check_exact,
+            check_categorical=check_categorical,
+            obj=f"{obj}.index",
+        )
```

Comment:
This is a cudf function, do we need to dispatch here?

Reply:
Sorry missed this one. Fixed it.